Slim/Menu/BrowseLibrary.pm

```diff
@@ -1555,15 +1555,71 @@ sub _albums {
 			}
 
 			my $extra;
+
+			# Have we got tracks for the year from other albums (ones with a different album year)?
+			my $yearTracks = 0;
+			my $year = (grep(/^year:/, @searchTags))[0];
+			if ( $year && !(grep(/^release_type:|^work_id:/, @searchTags)) ) {
+				$year =~ s/^year://;
+				my $lib = (grep(/^library_id:/, @searchTags))[0];
```

Comment:
Don't we already have this in `$library_id`?

Reply:
Indeed we do!